From: nabijaczleweli <nabijaczleweli@gmail.com>
Date: Thu, 20 Feb 2020 23:08:30 +0100
Subject: Don't build httplz

This avoids doing the build of the http binary twice,
and we hardlink httplz to http during the build anyway
---
 Cargo.toml | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/Cargo.toml b/Cargo.toml
index 03151c3..74cffac 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -75,12 +75,6 @@ path = "src/main.rs"
 test = false
 doc = false
 
-[[bin]]
-name = "httplz"
-path = "src/main.rs"
-test = false
-doc = false
-
 
 [package.metadata.deb]
 name = "http"
